sdk/eventhub/event-hubs/test/misc.spec.ts

```diff
@@ -132,129 +165,170 @@ describe("Misc tests", function(): void {
       "some string"
     ];
     const obj: EventData = { body: msgBody, properties: { message_id: uuid() } };
-    const offset = (await client.getPartitionProperties(partitionId)).lastEnqueuedOffset;
-    debug(`Partition ${partitionId} has last message with offset ${offset}.`);
+    debug(`Partition ${partitionId} has last message with offset ${lastEnqueuedOffset}.`);
     debug("Sending one message %O", obj);
-    receiver = client.createConsumer(EventHubConsumerClient.defaultConsumerGroupName, partitionId, {
-      offset
-    });
     await producerClient.sendBatch([obj], { partitionId });
     debug("Successfully sent the large message.");
-    const data = await receiver.receiveBatch(1, 30);
-    await receiver.close();
-    debug("received message: ", data);
-    should.exist(data);
-    should.equal(data.length, 1);
-    debug("Received message: %O", data);
-    assert.deepEqual(data[0].body, msgBody);
-    assert.strictEqual(data[0].properties!.message_id, obj.properties!.message_id);
+
+    let subscription: Subscription | undefined;
```

Comment:
Looks like we've caught onto a fairly common pattern - can you refactor this into a function?

Reply:
Well sure, we might as well write a receiveBatch helper method!

Reply:
I was thinking about the same, but there was this part in the new testing guidelines that we should avoid using helpers in tests as that adds 1 more level of indirection when understanding what a test does...

Reply:
Oo, that's a guideline I didn't know about! I get where that's coming from (the service bus tests have abstracted away nearly all meaning for many of the tests...)

Reply:
Also, some of the tests deal with batching where we need to collect the messages and do asserts on the outside. Others have the asserts from the inside. So, instead of shoehorning all types into a single helper, I chose to keep them separate